app/javascript/dashboard/store/modules/helpCenterPortals/actions.js

```diff
@@ -36,16 +36,15 @@ export const actions = {
     }
   },
 
-  update: async ({ commit }, { portalObj }) => {
-    const portalSlug = portalObj.slug;
+  update: async ({ commit }, { portalSlug, portalObj }) => {
```

Comment:
I think we don't need this since we are not passing slug.
```suggestion
  update: async ({ commit }, { portalObj }) => {
```

Reply:
Portal slug is mandatory for updating portal.

Reply:
Yes, removing this reverting back these changes. It is resolved, by the latest commit added the slug while sending an updated request.